utils/datasets.py

```diff
@@ -142,6 +149,7 @@ class DSprites(DisentangledDataset):
     lat_names = ('color', 'shape', 'scale', 'orientation', 'posX', 'posY')
     lat_sizes = np.array([1, 3, 6, 40, 32, 32])
     img_size = (1, 64, 64)
+    background_color = 0
```

Comment:
What is zero? Can we define c_black as 0 and then write background_color = c_black for example?

Reply:
Yeah I can do this

Reply:
yeah agree with bart just use BLACK = 0 at the top of the file and WHITE = 1